lte/gateway/python/integ_tests/s1aptests/test_services_are_running.py

```diff
@@ -0,0 +1,119 @@
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import time
+import unittest
+
+import yaml
+from integ_tests.s1aptests import s1ap_wrapper
+from s1ap_utils import InitMode, MagmadUtil
+from yaml.loader import SafeLoader
+
+SERVICE_START_TIME = 'start_time'
+SERVICE_ACTIVE = 'active_status'
+SERVICE_RESULT = 'result'
+
+
+class TestServicesAreRunning(unittest.TestCase):
+    """
+    A simple smoke test that checks if all relevant magma services are running.
+    This test was introduced because of GH14007 where a service that is not used
+    in the integration tests was damaged in the magma debian package artifact.
+    """
+
+    def setUp(self):
+        """Initialize"""
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper(
+            health_service=MagmadUtil.health_service_cmds.ENABLE,
+        )
+
+    def test_all_services_are_running(self):
```

Comment:
We were following the guidelines where file-name/class-name/test-function-name should match exactly for sake of easy debugging. Recommended:
`test_all_services_are_running` => `test_services_are_running`

Reply:
Thank you - test is renamed.